op-e2e/system_test.go

```diff
@@ -851,6 +856,11 @@ func TestFees(t *testing.T) {
 	}
 
 	cfg := DefaultSystemConfig(t)
+	// TODO: after we have the system config contract and new op-geth L1 cost utils,
+	// we can pull in l1 costs into every e2e test and account for it in assertions easily etc.
+	cfg.DeployConfig.GasPriceOracleOverhead = 2100
+	cfg.DeployConfig.GasPriceOracleScalar = 1000_000
+	cfg.DeployConfig.GasPriceOracleDecimals = 6
```

Comment:
I feel like the decimals should no longer be a config param and we should make it into a protocol constant, it could go in `protocol_params.go` in `op-geth` or live in a constants in `op-node`. I believe this config previously would set the storage in the gas oracle contract but now the decimals value is a `constant` so its inlined in the bytecode and not in storage

Reply:
Later PRs in the stack get rid of it, here I just try to minimize config changes.